protocol/0044-LIME-lp_mechanics.md

```diff
@@ -19,87 +59,81 @@ Any Vega participant can apply to become a liquidity provider (LP) on a market b
+
+(*) The expection is the end of the opening auction of the market. The LPs that submit commitment during the opening auction become the market LPs as soon as the opening auction ends.
+
+The fee for the market is only [updated at the epoch boundary using the "auction" mechanism set here](0042-LIQF-setting_fees_and_rewarding_lps.md). 
 
 Liquidity provider bond account:
 
 - Each active market has one bond account per liquidity provider, per settlement asset for that market.
 - When a liquidity provider transaction is approved, the size of their staked bond is immediately transferred from their general account to this bond account.
-- A liquidity provider can only prompt a transfer of funds to or from this account by submitting a valid transaction to create, increase, or decrease their commitment to the market, which must be validated and pass all checks (e.g. including those around minimum liquidity commitment required, when trying to reduce commitment).
+- A liquidity provider can only prompt a transfer of funds to or from this account by (re)submitting the LP commitment  transaction: a valid transaction to create, increase, or decrease their commitment to the market. 
   - Transfers to/from this account also occur when it is used for settlement or margin shortfall, when penalties are applied, and if the account is under-collateralised because of these uses and is subsequently topped up to the commitment amount during collateral search (see below)
 - Collateral withdrawn from this account may only be transferred to either:
   - The insurance pool of the market (in event of penalties/slashing)
   - The liquidity provider's margin account or the network's settlement account/other participant's margin accounts (during a margin search and mark to market settlement) in the event that they have zero balance in their general account.
   - The liquidity provider's general account (in event of liquidity provider reducing their commitment)
 
-### liquidity provider proposes to amend commitment amount
+### Liquidity provider proposes to amend commitment amount
 
 The commitment transaction is also used to amend any aspect of their liquidity provision obligations.
 A participant may apply to amend their commitment amount by submitting a transaction for the market with a revised commitment amount.
 
 `proposed-commitment-variation = new-proposed-commitment-amount - old-commitment-amount`
 
-#### INCREASING COMMITMENT
+The amendment is actioned at the beginning of the next epoch (after the rewards / penalties for present LPs - including the party that's amending - have been evaluated). It's the amending LP's responsibility to have sufficient balance in their general account at the epoch boundary; if not the LP amendment is not actioned.
```

Comment:
```suggestion
The amendment is actioned in two steps. 
1) the amount is immediately transferred from the party's general account to a temporary "intra-epoch" bond account. This amount counts towards the stake committed to the market and so in particular can get the market out of liquidity auction. 
2) at the beginning of the next epoch (after the rewards / penalties for present LPs - including the party that's amending - have been evaluated) the amount is transferred from the "intra-epoch" bond to the true bond account. ```

Reply:
Do they earn fees on this? I would assume they must otherwise why would someone do it, but then they could presumably increase them decrease again just before the epoch ends?

Reply:
This also could be a pain to track if someone added multiple times in an epoch. But I can't think of much else to allow intra-epoch adding without some sort of secondary account, hmm

Reply:
Thinking more, it might just work allowing increases whenever and decreases only at epoch end.

- Liquidity fees themselves are calculated and assigned at time of each trade, so you don't get a claim over historic fees in the epoch by submitting later
- SLA is based on your time on the book. We should ensure that if someone places their first commitment mid way through the block it counts as them having met their SLA perfectly until that point.
- Bonus payments are weighted by total fees received, so increasing later in the epoch should make that proportionally smaller. However it means your performance score is much more likely to be perfect, so I'm not quite sure how that ends up playing off against each other.